homework/src/App.js

```diff
@@ -17,12 +17,29 @@ class App {
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
     const root = document.getElementById('root');
+
+    const header = Util.createAndAppend('header', root);
+    Util.createAndAppend('h3', header, { 'html': 'HYF Repositories' })
+    const select = Util.createAndAppend('select', header);
+    Util.createAndAppend('div', root, { id: 'container' });
+
     // ...
 
     try {
       // ...
       const repos = await Util.fetchJSON(url);
+      repos.sort((a, b) => a.name.localeCompare(b.name, { ignorePunctuation: true }));
       this.repos = repos.map(repo => new Repository(repo));
+
+
+      repos.forEach((repo, index) => {
+        Util.createAndAppend('option', select, { 'html': repo.name, 'value': index });
+        this.fetchContributorsAndRender(0);
+        select.addEventListener('change', (e) => {
+          const index = e.target.value;
+          this.fetchContributorsAndRender(index);
+        });
+      });
```

Comment:
I may have overlooked this in my previous review, but lines 37-41 should not be in the `forEach` loop but come after it.

Reply:
One other thing I noticed: your app is not responsive:

![samtabanja](https://user-images.githubusercontent.com/2788771/45475621-34b58700-b73d-11e8-91b5-5870795328fe.png)

Reply:
thanks dear , ill work on it  and ill make it responsive asap.